src/cli/arguments.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2019 Luxoft Sweden AB
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#include "cli/arguments.h"
+
+#include <glibmm.h>
+
+namespace UserIdentificationManager::Cli
+{
+    std::optional<Arguments> Arguments::parse(int argc, char *argv[], std::ostream &output)
+    {
+        Arguments arguments;
+        Glib::OptionGroup main_group("main", "Main Options");
+        Glib::OptionContext context;
+
+        {
+            Glib::OptionEntry entry;
+            entry.set_long_name("version");
+            entry.set_description("Print version and exit");
+            main_group.add_entry(entry, arguments.print_version_and_exit);
+        }
+
+        {
+            Glib::OptionEntry entry;
+            entry.set_short_name('i');
+            entry.set_long_name("identified-users");
+            entry.set_description("Print recently identified users");
```

Comment:
Consider add suffix ".. since start of daemon" or something similar

Reply:
Maybe do both. Make it clear that it is since start of daemon and just the X most recent ones. 

Hum, I wonder if I should just hardcode # users here in the string or have a constant for it that is shared between client and daemon. Not sure it is worth the hassle... but error prone to not have it be shared. Will try to share.

Reply:
It now reads "Print users identified since start of daemon (max 20)".